ingestion/src/metadata/ingestion/source/dashboard/looker/connection.py

```diff
@@ -61,8 +61,17 @@ def list_datamodels_test():
         """
         assert client.all_lookml_models(limit=1)
 
+    def validate_api_version():
+        """
+        Make sure we get a True
+        """
+        assert "4.0" in (
```

Comment:
does it have to be 4.0 specifically or it could be 4.1, etc.

Reply:
we are using the sdk for 4.0 so feels safe to stick with this. If they update it we might need to change as well. https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/src/metadata/ingestion/source/dashboard/looker/connection.py#L44